site/src/helpers/planner.ts

```diff
@@ -137,12 +137,15 @@ export const loadRoadmap = async (
       roadmap = request.data.roadmap;
     }
   }
-  // check local storage next
+
+  let isLocalNewer = false;
+
   if (!roadmap && localRoadmap) {
-    roadmap = JSON.parse(localRoadmap);
-  }
-  // no saved planner
-  if (!roadmap) {
+    roadmap = localRoadmap;
+  } else if (roadmap && localRoadmap && (localRoadmap.timestamp ?? 0) > (roadmap.timestamp ?? 0)) {
```

Comment:
I suppose this logic would work on prod instances, but it was confusing that it didn't work locally because a different branch didn't set the timestamp of the roadmap. I think this PR in itself is a pretty rare occurrence though, and for those cases this code would work

Reply:
I'm not sure what you mean. It seemed to work on my initial tests locally and I nullish coalesced the timestamps with 0 to try to avoid issues with missing timestamps on old schedules.

Reply:
I think it was just an issue for some branches without the code, so you can disregard this thread.